src/Umbraco.Web.UI.Client/src/packages/multi-url-picker/link-picker-modal/link-picker-modal.element.ts

```diff
@@ -262,14 +263,30 @@ export class UmbLinkPickerModalElement extends UmbModalBaseElement<UmbLinkPicker
 		this.#partialUpdateLink({ type: 'external' });
 	}
 
-	#checkIfUrlIsMissing() {
-		if (this.value.link.type !== 'external') return false;
-		const hasUrl = this.value.link.url && this.value.link.url.length > 0;
-		const hasAnchor = this.value.link.queryString && this.value.link.queryString.length > 0;
+	#validateExternalLink(): void {
+		if (this.#isNonExternalType()) return;
+
+		this._missingType = this.#isUrlMissing();
```

Comment:
This seems like bad practice, this property is being set by the validator, should not be overwritten by other logic, this will make the code very hard to maintain in the future. For now I will just make the submit button disable when there is no type